service/src/integrationTest/kotlin/fi/espoo/evaka/pis/InactiveEmployeesRoleResetIntegrationTest.kt

```diff
@@ -0,0 +1,150 @@
+package fi.espoo.evaka.pis
+
+import fi.espoo.evaka.PureJdbiTest
+import fi.espoo.evaka.application.utils.helsinkiZone
+import fi.espoo.evaka.resetDatabase
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.auth.insertDaycareAclRow
+import fi.espoo.evaka.shared.dev.DevCareArea
+import fi.espoo.evaka.shared.dev.DevDaycare
+import fi.espoo.evaka.shared.dev.DevEmployee
+import fi.espoo.evaka.shared.dev.insertTestCareArea
+import fi.espoo.evaka.shared.dev.insertTestDaycare
+import fi.espoo.evaka.shared.dev.insertTestEmployee
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDateTime
+import java.time.ZonedDateTime
+import java.time.temporal.ChronoUnit
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class InactiveEmployeesRoleResetIntegrationTest : PureJdbiTest() {
+    private val FirstOfAugust2021 = ZonedDateTime.of(LocalDateTime.of(2021, 8, 1, 3, 15), helsinkiZone).toInstant()
```

Comment:
`private val FirstOfAugust2021` vs `private val firstOfAugust2021`?

Reply:
Thanks, I was so focused on capitalizing the months that I accidentally capitalized everything 😄